ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-nightly.yaml

```diff
@@ -3506,6 +3506,27 @@ tests:
       TAG_VERSION: '@4.15'
     test:
     - chain: openshift-e2e-test-qe-ui
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28
+  cron: 28 20 15 * *
+  steps:
+    cluster_profile: vsphere
+    env:
+      E2E_RUN_TAGS: '@vsphere-ipi and @network-ovnkubernetes and not @fips'
+      IP_FAMILIES: DualStackIPv6Primary
+      TAG_VERSION: '@4.15'
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-vsphere-ipi-ovn-dualstack
+- as: vsphere-ipi-ovn-dualstack-privmaryv6-f28-destructive
+  cron: 22 23 17 * *
```

Comment:
pls generate a random cron item by `ci-operator/config/openshift/openshift-tests-private/tools/generate-cron-entry.sh` to avoid concurrent running jobs.

Reply:
done